src/app/first-time-login-dialog/first-time-login-dialog.component.ts

```diff
@@ -39,6 +39,9 @@ export class FirstTimeLoginDialogComponent implements OnInit, OnDestroy {
   ) {}
 
   ngOnInit(): void {
+    if (window.location.pathname.includes('/invitations/')) {
```

Comment:
You should use the router to get the current path, e.g. https://angular.io/api/router/ActivatedRouteSnapshot

Reply:
I tried to, it didn't work. The route snapshot was completely empty at the point where it was called...

Reply:
Actually, we will rework the first-time-dialog in https://github.com/appuio/cloud-portal/issues/524 soon, so for  the time being I consider this good enough.